.github/workflows/publish-docker.yml

```diff
@@ -0,0 +1,77 @@
+name: Publish Docker
+
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: tag (ex. v0.8.3) to publish on docker
+        required: true
+
+jobs:
+  tag-docker:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        image: ["tanssi", "container-chain-simple-template", "container-chain-evm-template"]
```

Comment:
you can use the include syntax to make refactoring a bit easier if you ever need to change names or targets:

```
 include:
          - image: "tanssi"
            files: ["tanssi-node", "tanssi-node-skylake", "tanssi-node-znver3"]
          - image: "container-chain-simple-template"
            files: ["container-chain-template-simple-node", "container-chain-template-simple-node-skylake", "container-chain-template-simple-node-znver3"]
          - image: "container-chain-evm-template"
            files: ["container-chain-template-frontier-node", "container-chain-template-frontier-node-skylake", "container-chain-template-frontier-node-znver3"]
```